src/Diagnostics.ModelsAndUtils/DataTableUtility.cs

```diff
@@ -0,0 +1,73 @@
+﻿using Diagnostics.ModelsAndUtils;
+using Diagnostics.ModelsAndUtils.Models;
+using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+
+namespace Diagnostics.ModelsAndUtils
+{
+    public static class DataTableUtility
+    {
+        public static DataTable GetDataTable(DataTableResponseObject dataTableResponse)
```

Comment:
Instead of creating this as methods, do you think it will nice to create extensions for DataTableResponseObject.

Like "res.ToDataTable()"

Reply:
I think this would be fine. These specific methods are not supposed to be used by the script authors, as we use it inside kusto data provider to go from DataTableResponseObject => DataTable and then later after the script runs to go the opposite transformation. 

I can create extensions for DataTableResponseObject/DataTable that will do these transformations